KINESIS.md

```diff
@@ -0,0 +1,49 @@
+# Development Environment Setup
+
+- [Download & install vscode](https://code.visualstudio.com/download)
+- Open the kinesis-core project in vscode and click on "Reopen in Container" when asked
+![image](https://user-images.githubusercontent.com/29750/203445568-939211f6-126f-4150-8b7e-d2b3360effff.png)
+
+- Install vscode extensions inside the remote container once it is opened, install the following extensions:
+    * [C++ Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-extension-pack)
+    * [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)
```

Comment:
Add these recommended extensions to `.vscode/extenions.json` 
```json
{
  "recommendations": [EXENDION_NAME(s)]
}
```

Reply:
fixed in https://github.com/bullioncapital/kinesis-core/pull/24/commits/5dc397aac8c33bea463a20252b401f7341e7c36c